Edited release date and team members

mrkellysch opened the pull request:
Funding and release dates are dependant on many factors.  Team members need to be made accessable so that new members know who's running Chia

mrkellysch committed:
Edited release date and team members

Funding and release dates are dependant on many factors.  Team members need to be made accessable so that new members know who's running Chia

alfondotnet commented on sh.exe.stackdump:
```diff
@@ -0,0 +1,8 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FFC601133B7
```
should we remove this from the PR?

alfondotnet commented on webroot/faq/faq.md:
```diff
@@ -150,6 +150,11 @@ A: Instead of just one whitepaper, we're going to have academic papers in journa
 
++ <b>Q: Who is in the Chia team?</b>
+
+A: Bram Cohen: @bramcohen, Ryan Singer: @ryan_singer , Richard Kiss: @richardkiss and Alex Wice: @alexwice 
```
These are keybase handles, I feel like this could be noted, or in case these are supposed to be Twitter handles, include a link, considering:

The twitter handle  for Alex Wice is `@awice`
The twitter handle for Ryan Singer is `@rjs`

mrkellysch replied:
Spoke to Ryan Singer about updating the homepage with Team, members, their titles and their Keybase handles. You may want to add Twitter handles, but I presume you want communication about Chia on Keybase. This decision is up to the team. Just thought it was important that interested parties could see who members were so that they could appreciate any announcements of Keybase chat as being from an employee / founder of Chia.

alfondotnet replied:
That sounds right!. What I meant is that for general public though, it might seem these are Twitter handles, so a clarification in this regard would I think be good :)

mrkellysch committed:
Clarified mimimum hard drive space

@memorysaver said "I am sorry that I don't quite undertand the technique behind the scene to translate this. Is anyone can help me to have a better understanding ?".

I noticed this was quite tecnical and could do with some elaboration. Thanks to @bramcohen.

ashadle commented:
@mrkellysch spoke with Bram and we don't want to include the team on our website for now.  There are also merge conflicts on this page so I'm gong to close this PR.
